src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task {
+    protected String deadline;
+
+    public Deadline(String description, String deadline) {
+        super(description);
+        deadline = deadline.replace("by", "by:");
```

Comment:
I noticed that `deadline` here is also recycled, perhaps consider my recommendation in the prior comment on `Ava.java` from line 64 to 73?